### PR TITLE
Add a step which logs kubelet logs to console

### PR DESCRIPTION
**Pull request by vrutkovs**
This should help us debug issues where the machine starts but doesn't join as a node. Kubelet would post logs to console and `gather-*-console` step would fetch the latest output.

Example console output: https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/23681/rehearse-23681-pull-ci-red-hat-storage-odf-operator-release-4.9-odf-operator-bundle-e2e-aws/1461344672763875328/artifacts/odf-operator-bundle-e2e-aws/gather-aws-console/artifacts/i-030afccf539b24521

### CODE REVIEW

File: ci-operator/step-registry/openshift/manifests/kubelet-logs-to-console/openshift-manifests-kubelet-logs-to-console-ref.metadata.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+	"path": "openshift/manifests/kubelet-logs-to-console/openshift-manifests-kubelet-logs-to-console-ref.yaml",
+	"owners": {
+		"approvers": [
+			"smarterclayton",
+			"wking",
+			"vrutkovs",
+			"abhinavdahiya",
```
Comment (dgoodwin) [inline ci-operator/step-registry/openshift/manifests/kubelet-logs-to-console/openshift-manifests-kubelet-logs-to-console-ref.metadata.json:8]: No longer with the company fwiw.

**Review by dgoodwin — COMMENTED** [review]
This is cool and theoretically should help, though probably not with the problem we're facing right now which is Azure specific.

We were working on a step to gather azure boot log (console) https://gist.github.com/dgoodwin/05f2babfae52a207d5541e1c7d3593b6 but ran into a blocker where apparently, workers don't get boot diagnostics enabled, whereas control plane machines do.

We abandoned that approach as a result and instead did https://github.com/openshift/release/pull/23640 which uses the pre-existing ssh bastion support in CI, and then fetched full system journal.

None the less this might still be helpful for others. Or we could invest more heavily into the ssh bastion approach instead and go for full journals?

File: ci-operator/step-registry/openshift/manifests/kubelet-logs-to-console/openshift-manifests-kubelet-logs-to-console-ref.metadata.json (hunk 1 above)
Comment (vrutkovs) [reply ci-operator/step-registry/openshift/manifests/kubelet-logs-to-console/openshift-manifests-kubelet-logs-to-console-ref.metadata.json:8]: That file is a symlink from manifests, so we'll need a PR to clean up all instances

**Pull request closed without merging**